lib/blocks.php

```diff
@@ -232,7 +232,7 @@ function do_blocks( $content ) {
 			$inner_content = '';
 
 			if ( ! $is_self_closing ) {
-				$end_tag_pattern = '/<!--\s+\/wp:' . str_replace( '/', '\/', preg_quote( $block_name ) ) . '\s+-->/';
+				$end_tag_pattern = '/<!--\s+\/wp:' . preg_quote( $block_name, '/' ) . '\s+-->/';
```

Comment:
this is a great change. but…while we're at it, what if we changed the delimiter too so remove a tiny bit of escaping?

```php
'~<!--\s+/wp:' . preg_quote( $block_name, '~' ) . '\s+-->~'
```

just a thought, but it only saves one escape 🤷‍♂️